src/lib/FormGroupLayout.tsx

```diff
@@ -44,7 +45,7 @@ const FormGroupLayout = <T extends FieldValues>(props: FormGroupLayoutProps<T>)
         <InputGroup
           style={{
             flexWrap: "nowrap",
-            alignItems: "center",
```

Comment:
did you delete this on purpose? 

Is it not needed anymore?

Reply:
We need to specify different style if we want to display the `InputGroupText` or a specific component

Reply:
@severinoAmmirati but I think in that case it makes more sense to leave the `alignItems center` in there and if somebody needs another style he can set `alignItems` to something else with the `inputGroupStyle`

Reply:
@neotrow `alignItems` restored